src/components/Requirements/CompletedSubReqCourse.vue

```diff
@@ -37,94 +37,55 @@
 import Vue, { PropType } from 'vue';
 import ReqCourse from '@/components/Requirements/ReqCourse.vue';
 import store from '@/store';
+import getCurrentSeason, { getCurrentYear } from '@/utilities';
 
-type CompletedSubReq = {
-  color: string;
-  transferCreditColor: string;
-  courseSubject: string;
-  courseNumber: string;
-  courseUniqueId: number;
-  isTransferCredit: boolean;
-  semesterType: FirestoreSemesterType;
-  semesterYear: number;
-};
+const transferCreditColor = 'DA4A4A'; // Arbitrary color for transfer credit
 
 export default Vue.extend({
   components: { ReqCourse },
   props: {
     subReqCourseId: { type: Number, required: true },
-    crsesTaken: { type: Array as PropType<readonly CourseTaken[]>, required: true },
-  },
-  data(): CompletedSubReq {
-    return {
-      semesterType: 'Fall', // 'Fall' is an arbitrary FirestoreSemesterType value that user will never see
-      semesterYear: 0,
-      color: '',
-      transferCreditColor: 'DA4A4A', // Arbitrary color for transfer credit
-      courseSubject: '',
-      courseNumber: '',
-      courseUniqueId: 0,
```

Comment:
If it's transfer, keep it 0, if it's not, then we read from the `courseTaken` object.